common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisInteriorManager.java

```diff
@@ -79,7 +79,7 @@ public ProtectedZone[] unbreakableZones() {
 
         ProtectedZone ctrlRoomAirlck = new ProtectedZone(corridorAirlockCenter.below(2).north(2).west(3), corridorAirlockCenter.south(3).east(3).above(6), "control_room_airlock");
         ProtectedZone hubAirlck = new ProtectedZone(STATIC_CORRIDOR_POSITION.below(2).north(2).west(3), STATIC_CORRIDOR_POSITION.south(3).east(3).above(6), "hub_airlock");
-        ProtectedZone arsRoom = new ProtectedZone(new BlockPos(1009, 97, -2), new BlockPos(1041, 118, 30), "ars_room");
+        ProtectedZone arsRoom = new ProtectedZone(new BlockPos(1051, 97, 6), new BlockPos(1023, 118, 36), "ars_room");
```

Comment:
We should should make all of these constants

Reply:
This needs to use the newly added ARS_TREE_CORNER_A and ARS_TREE_CORNER_B constants for consistency.